controllers/subscriber/subscriberControllers.js

```diff
@@ -153,21 +153,30 @@ export class IdSubscriberController extends BaseSubscriber {
 		super(host, name, options);
 
 		this._idPropertyName = options && options.idPropertyName;
-		this._idPropertyValue = this._idPropertyName ? this._host[this._idPropertyName] : undefined;
 		this._registries = new Map();
 		this._registryControllers = new Map();
+
+		this._handlePassthrough = this._handlePassthrough.bind(this);
 	}
 
 	get registries() {
 		return Array.from(this._registries.values());
 	}
 
+	hostConnected() {
+		// Id controllers will catch events the same name of nested children and hook them up to the same registry(ies?)
+		if (this._eventName) this._host.addEventListener(this._eventName, this._handlePassthrough);
+	}
+
 	hostDisconnected() {
 		super.hostDisconnected();
+		if (this._eventName) this._host.removeEventListener(this._eventName, this._handlePassthrough);
+
 		if (this._registryObserver) this._registryObserver.disconnect();
 		this._registryControllers.forEach(controller => {
 			controller.unsubscribe(this._host);
 		});
+		this._idPropertyValue = undefined;
```

Comment:
I think this is a current bug with disconnected then reconnect IdSubscribers not being able to re-register.